File: 242-valid-anagram/valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t)
    {
        sort(s.begin(), s.end());
        sort(t.begin(), t.end());
        if (s.size() != t.size())
            return false;
        int i = 0;
        while (s[i])
        {
            if (s[i] != t[i])
                return (false);
            i++;
        }
        return (true);
        
    }
};

File: 242-valid-anagram/valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        mymap1 = {}
        mymap2 = {}
        for i in range(len(s)):
            mymap1[s[i]]  = mymap1.get(s[i], 0) + 1
        for i in range(len(t)):
            mymap2[t[i]] =  mymap2.get(t[i], 0) + 1
        return mymap1 == mymap2        